Assets/Scripts/Layers/Adapters/DataTypeAdapters/NL3DImportAdapter.cs

```diff
@@ -0,0 +1,26 @@
+using Netherlands3D.Twin.Projects;
+using UnityEngine;
+
+namespace Netherlands3D.Twin
+{
+    [CreateAssetMenu(menuName = "Netherlands3D/Adapters/NL3DImportAdapter", fileName = "NL3DImportAdapter", order = 0)]
+    public class NL3DImportAdapter : ScriptableObject, IDataTypeAdapter
+    {
+        public bool Supports(LocalFile localFile)
+        {
+            return localFile.SourceUrl.ToLower().EndsWith(".nl3d");      
```

Comment:
Nice example of where we do not need the body of the file and it can -presumably- be large - as mentioned earlier I have serious doubts whether we should always download the file contents

Reply:
Download/dont download at start will be a seperate ticket